src/_pytest/capture.py

```diff
@@ -512,7 +532,11 @@ class NoCapture:
     __init__ = start = done = suspend = resume = lambda *args: None
 
 
-class FDCaptureBinary:
+class Capture:
+    pass
```

Comment:
This empty class is here for type hinting only? Perhaps adding a quick one line docstring mentioning that?

Reply:
Yeah, or remove it.
This certainly needs another pass.
I've thought about pulling out the renaming/moving of the new classes only for now, given that combinations are not really needed currently anyway (also because of the "choices" validation).

It might make sense to pick that part into #6690, but there it appears to be unclear still how to do it (at least to me).  Will try/look into that later, but might take some time.

Reply:
OK, thanks